### PR TITLE
Bug 2049576: making sure cancelled rollback behaves as epected

### PR DESCRIPTION
**Pull request by JaydipGabani**
Fixes [this](https://bugzilla.redhat.com/show_bug.cgi?id=2049576) bug

### CODE REVIEW

**Comment by eriknelson** [comment]
My bad, somehow that bz was missing a target release but I had it marked triaged. Updated!

File: src/app/plan/duck/selectors.ts [hunk 1]
```diff
@@ -154,6 +158,8 @@ const getPlansWithPlanStatus = createSelector(
       } else {
         planMigrations = plan.Migrations;
       }
+      console.log('migration');
```
Comment (ibolton336) [inline src/app/plan/duck/selectors.ts:161]: Can we remove these?

Comment (JaydipGabani) [reply src/app/plan/duck/selectors.ts]: Done

**Review by mturley — APPROVED** [review]
LGTM, good catch

**Pull request merged**